web/samples/chirality/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    
+    <!-- Set the viewport width to match phone and tablet device widths -->                         
+    <meta name="viewport" content="width=device-width,user-scalable=no" /> 
+
+    <!-- Allow KeymanWeb to be saved to the iPhone home screen -->
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    
+    <!-- Enable IE9 Standards mode -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" /> 
+      
+    <title>KeymanWeb Sample Page - Uncompiled Source</title>
+
+    <!-- Your page CSS --> 
+    <style type='text/css'>   
+      body {font-family: Tahoma,helvetica;}
+      h3 {font-size: 1em;font-weight:normal;color: darkred; margin-bottom: 4px}
+      .test {font-size: 1.5em; width:80%; min-height:30px; border: 1px solid gray;}
+      #KeymanWebControl {width:50%;min-width:600px;}       
+    </style> 
+ 
+    <!-- Insert uncompiled KeymanWeb source scripts -->              
+    <script src="../../source/kmwstring.js" type="application/javascript"></script>
+    <script src="../../source/kmwbase.js" type="application/javascript"></script>
+    <script src="../../source/keymanweb.js" type="application/javascript"></script>
+    <script src="../../source/kmwosk.js" type="application/javascript"></script>
+    <script src="../../source/kmwnative.js" type="application/javascript"></script>
+    <script src="../../source/kmwcallback.js" type="application/javascript"></script>
+    <script src="../../source/kmwkeymaps.js" type="application/javascript"></script>
+    <script src="../../source/kmwlayout.js" type="application/javascript"></script>
+    <script src="../../source/kmwinit.js" type="application/javascript"></script>             
+
+    <!-- 
+      For desktop browsers, a script for the user interface must be inserted here.
+       
+      Standard UIs are toggle, button, float and toolbar.  
+      The toolbar UI is best for any page designed to support keyboards for 
+      a large number of languages.
+    -->
+    <script src="../../source/kmwuitoggle.js"></script>                                                                                           
+ 
+    <!-- Initialization: set paths to keyboards, resources and fonts as required -->
+    <script>
+  // Thank you to https://stackoverflow.com/questions/22607150/getting-the-url-parameters-inside-the-html-page. 
+  var GetURLParameter = function(sParam) {
+    var sPageURL = window.location.search.substring(1);
+    var sURLVariables = sPageURL.split('&');
+    for (var i = 0; i < sURLVariables.length; i++) {
+      var sParameterName = sURLVariables[i].split('=');
+      if (sParameterName[0] == sParam) {
+        return sParameterName[1];
+      }
+    }
+  };
+  
+  var attachType = GetURLParameter("mode");
```

Comment:
The lines involving `attach... ` seem to be from samples/attachment-api, and can probably be removed from the chirality test page. (simplify)

Reply:
Ah, the dangers of CTRL+C CTRL+V.